src/components/TableOfContents.tsx

```diff
@@ -258,14 +77,68 @@ const TableOfContentsLink: React.FC<IPropsTableOfContentsLink> = ({
   if (isNested) {
     classes += " nested"
   }
+
+  const $dotBg = cssVar("dot-bg")
```

Comment:
Could you explain a bit about `cssVar`? how does this work?
Do you define it on `[$dotBg.variable]: "colors.background"` and then you can use the reference of that?

And wondering why you choose that name for it, shouldn't this be named `linkBg`?

Reply:
You have the correct idea! `cssVar` takes a string to name the variable, and returns two strings to use:
1. The variable declaration (`--dot-bg`) as `$dotBg.variable` when setting a value to it. (i.e. `[$dotBg.variable]: <someValue>`)
2. The variable call (`var(--dot-bg)`)  as `$dotBg.reference` when retrieving the value (i.e. `backgroundColor: $dotBg.reference`)
> Aside: you can also supply a fallback value as a second argument, which is the same as the fallback in the `var()` function.

When calling a token name for CSS variables, you have to start with the name of the theme collection it belongs to, hence `colors.someToken`

I chose `dot-bg` as the name because it is for the background color of the dot that is rendered off to the left when hovering over the link or if the link is active, and is not applied to the link itself. Could make it `link-dot-bg` to make it clearer.